libs/knowledge-store/ragstack_knowledge_store/graph_store.py

```diff
@@ -46,14 +50,52 @@ class SetupMode(Enum):
     ASYNC = 2
     OFF = 3
 
+def _serialize_metadata(md: Dict[str, Any]) -> str:
+    print("Serializing MD")
+    if isinstance(md.get("links"), Set):
+        print("Copying")
```

Comment:
remove print

Reply:
Heh. Can you tell that initially, this was failing to serialize and just hanging?